src/test/ui/inference/cannot-infer-async-enabled-impl-trait-bindings.stderr

```diff
@@ -13,7 +13,9 @@ error[E0282]: type annotations needed for `impl Future`
 LL |     let fut = async {
    |         --- consider giving `fut` the explicit type `impl Future`, with the type parameters specified
 LL |         make_unit()?;
-   |                    ^ cannot infer type
+   |                    ^ cannot infer type of `?` error
```

Comment:
I don't think "`?` error" is terminology used anywhere else, but I was looking for a friendly way to say `<_ as Try>::Error`.

Reply:
```suggestion
   |                    ^ cannot infer type of error for `?` operator
```
This isn't great either, but I think it's marginally better.